src/modules/fancyzones/FancyZonesLib/GuidUtils.h

```diff
@@ -16,7 +16,10 @@ namespace std
         size_t operator()(const GUID& Value) const
         {
             RPC_STATUS status = RPC_S_OK;
+            #pragma warning(push)
+            #pragma warning(disable : 26492)
```

Comment:
Why are we adding this to FancyZonesLib? Seems weird that we need to change FancyZones code.

Reply:
Thank you! 

(Had temporary issue compiling after VS update and forgot to remove. Now removed.)